src/main/java/com/google/api/gax/grpc/ApiCallable.java

```diff
@@ -105,10 +105,10 @@
    * to configure the service-level settings with.
    * @return {@link com.google.api.gax.grpc.ApiCallable} callable object.
    */
-  public static <RequestT, ResponseT> ApiCallable<RequestT, ResponseT> create(
+  public static <RequestT, ResponseT> BundlableApiCallableInfo<RequestT, ResponseT> create(
```

Comment:
Since the BundlingCallSettings was constructed before this create call, the BundlerFactory should be available at the caller site (from the BundlingCallSettings), and this function shouldn't have to return this type.